src/style.css

```diff
@@ -1,3 +1,80 @@
+
+
+html {
+  margin: 0;
+  padding: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+
+  --main-color: #000;
+  --blue-color: #00ACDC;
```

Comment:
`--blue-color` - is actually the main color, the variable names should show their purpose